.sphinx/requirements.txt

```diff
@@ -6,6 +6,6 @@ sphinx-tabs
 sphinx-reredirects
 pyspelling
 sphinxext-opengraph
-lxd-sphinx-extensions
+git+https://github.com/canonical/lxd-sphinx-extensions@refs/pull/26/merge#egg=lxd-sphinx-extensions
```

Comment:
Shouldn't we be using a doc-specific repo? It looks really odd to be seeing LXD in there. Seems to me we have a doc-tools repo somewhere.

Reply:
See https://github.com/canonical/lxd-sphinx-extensions/issues/25 - we need to rename the repo, but I want to check with @evildmp about the name first to avoid having to rename it again later.